stack/base.yml

```diff
@@ -0,0 +1,120 @@
+version: '3.7'
```

Comment:
You shouldn't add this new file here. Instead, fix the bug in this [file](https://github.com/CESARBR/knot-cloud/blob/master/stacks/core/base/base.yml).
Also, please verify if any other Mainflux's service depends on the jaeger to start.

Reply:
I have found another issue regarding the environment variables filepath. The connector.yml, connector.dev.yml inside the `core` folder are with incorrect env filepaths. I tried to create the services but it failed. My question is if I put the `env.d` folder in each of these file locations or I write one absolute filepath for all these YAML files.

Reply:
We don't need to guarantee this correctness to the `.yml` files stored in the `addons` since they are used as demand and the CLI will already copy the `connector` file to the right directory. Remember the CLI will be used only for development. Otherwise, when bringing these services to production, the developer will be responsible to locate the addon where necessary or to update the path accordingly.

Reply:
Done.